Overview.md

```diff
@@ -29,23 +29,79 @@ page_nav:
+- [Create and manage teams](insert link to Create a Team tutorial) that suit your organizational needs.
+- [Import calendars and meeting rooms](insert link to Import a Meeting tutorial) from external applications.
+- Connect to your Employee Management System for quick [team onboarding](insert link to Import a Team tutorial).
 
-| First Header  | Second Header |
-| ------------- | ------------- |
-| Content Cell  | Content Cell  |
-| Content Cell  | Content Cell  |
+**add appropriate links to the above list items for quick access to their associated docs or headings in this doc**
+
+Not sure where to begin? See the [getting started guide](insert link to Quick Start Guide) to learn how to import your first calendar.
+  
+## How Agenda works
+
+Agenda connects to external calendars and meeting platforms to merge user meeting details into one simplified view, providing a more comprehensive picture of team availability and simplifying the scheduling workflow. With everyone's meeting details available within one space, you can quickly create teams and schedule meetings that work well for all.
+
+You can manage Agenda's capabilities through the simplified [user interface](UI.md) or integrate Agenda with your external applications using the [RESTful API](API.md).
+
+### Quick scheduling
+
+Agenda's home view displays your upcoming meetings and provides quick access to the [scheduling tool](insert link to the Schedule a Meeting tutorial) in one simplified user interface. 
+
+![](images/overview-home-view.png)
+
+From here, you can select the plus icon to start scheduling a new meeting. Agenda's scheduling workflow walks you through all of the necessary steps to set up a meeting with your team members and suggests recommended times based on everyone's availability.
+
+### Organizing teams
+
+Agenda allows you to organize your people just as easily as your meeting schedules. You can [create and customize teams](insert link to Create a Team tutorial) to organize users based on project assignments, company team structure, or any other team assignment that reflects your organizational needs. Teams are searchable by various tag criteria to help you quickly locate and assign the right people you need in your meetings.
+
+![](images/overview-team-view.png)
+
+When scheduling team meetings, simply select the team and the availability of all its members is factored in to suggest the ideal meeting times that work best for everyone.
+
+### Merging external calendars
+
+Agenda enables simplified [connections to your external scheduling applications](insert link to Connect to a Source tutorial) from the Sources tab in the UI. You can choose from several scheduling applications to [merge your meeting information](insert link to Import a Meeting tutorial) into one space. 
+
+![](images/overview-sources-view.png)
+
+Agenda unifies this data to provide you, and your team members, with a clearer view of your availability. 
```

Comment:
Commas can be removed from this sentence, and should probably end with "...a clearer view of each other's availability."